docs/source/release_notes.rst

```diff
@@ -94,11 +94,11 @@ Version 0.0.1
 Notes
   * Initial beta release of IBM Z core collection, referred to as ibm_zos_core
```

Comment:
"IBM Z core collection" to "IBM z/OS core collection"

Reply:
ack